tests/MessageTest.php

```diff
@@ -271,6 +271,13 @@ public function testMessageBodySummaryWithSpecialUTF8Characters(): void
         self::assertSame('’é€௵ဪ‱', Psr7\Message::bodySummary($message));
     }
 
+    public function testMessageBodySummaryWithSpecialUTF8CharactersAndLargeBody(): void
```

Comment:
Is `AndLargeBody` appropriate here?

Reply:
I copied the naming scheme from the `testMessageBodySummaryWithLargeBody` above